index.js

```diff
@@ -169,6 +216,16 @@ const line = []
 
 
 // ***** Scope & Closures - Question 2 *****
+function ticketNumberGeneratorFunc() {
+  let ticketNumber = 0;
+
+  function takeATicketNumber(lineArray) {
+    ticketNumber += 1;
+    lineArray.push(ticketNumber);
+    return (`Welcome. You are ticket number ${ticketNumber}.`);
+  }
+  return takeATicketNumber
+}
```

Comment:
Excellent! Only small thing - there's no need for the parentheses in the return on line 225. This would work just fine:

```js
    return `Welcome. You are ticket number ${ticketNumber}.`;
```